src/Components/Body/Cards/Cad.tsx

```diff
@@ -14,6 +14,7 @@ import {
 import { AnimationOnScroll } from "react-animation-on-scroll";
 import { FC } from "react";
 
+// Cad card
```

Comment:
Again here the use of styled components could be used to make this code cleaner.

For example, if you extract the style for the Heading like so:

```tsx
const StyledHeading = styled(Heading)`
    size: "sm";
`;
```

you can reuse it everywhere like so:
```tsx
<StyledHeading> ... </StyledHeading>
...
<StyledHeading> ... </StyledHeading>
```